legend-engine-pure-code-compiled-core/src/main/resources/core/pure/serialization/toPureGrammar.pure

```diff
@@ -62,7 +62,9 @@ function <<access.private>> meta::pure::metamodel::serialization::grammar::remov
                                                  |  ^$fe(parametersValues=$fe.parametersValues->at(0)->meta::pure::metamodel::serialization::grammar::removeAutomap()->concatenate( $fe.parametersValues->tail()))
                                                 );
 
-                                           ,| ^$fe(parametersValues=$fe.parametersValues->at(0)->meta::pure::metamodel::serialization::grammar::removeAutomap()->concatenate( $fe.parametersValues->tail()))
+                                           ,| if ($fe.parametersValues->isNotEmpty(),
```

Comment:
we should avoid the if here, and do a head and a map

`^$fe(parametersValues=$fe.parametersValues->head()->map(p |  $p->meta::pure::metamodel::serialization::grammar::removeAutomap())->concatenate( $fe.parametersValues->tail()))`